Application/src/main/java/model/SegnalazioneDAO.java

```diff
@@ -0,0 +1,154 @@
+package model;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SegnalazioneDAO {
+	
+	public void save(Segnalazione segnalazione) throws SQLException {
+	    
+	    try (Connection connection = ConDB.getConnection();
+	         PreparedStatement preparedStatement = connection.prepareStatement(
+	                 "INSERT INTO segnalazione (motivazione, stato, utente_segnalato, utente_segnalante, ID_evento) VALUES (?, ?, ?, ?, ?)")) {
+
+	        // Imposta i parametri della query
+	        preparedStatement.setString(1, segnalazione.getMotivazione()); // Colonna motivazione
+	        preparedStatement.setString(2, "in attesa"); // Stato predefinito
```

Comment:
Per lo stato si dovrebbe comunque usare quello che sta nella segnalazione passata come parametro della funzione.

```suggestion
	        preparedStatement.setString(2, segnalazione.getStato()); // Colonna stato
```